addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js

```diff
@@ -178,12 +178,13 @@ export class ImageCrop extends Component {
         this.$cropperImage = this.$('.o_we_cropper_img');
         const cropperImage = this.$cropperImage[0];
         [cropperImage.style.width, cropperImage.style.height] = [this.$media.width() + 'px', this.$media.height() + 'px'];
-        
+
         const sel = this.document.getSelection();
         sel && sel.removeAllRanges();
 
         // Overlaying the cropper image over the real image
-        const offset = this.$media.offset();
+        const mediaRect = this.media.getBoundingClientRect();
+        const offset = { left: mediaRect.left, top: mediaRect.top };
```

Comment:
The fix seems ok, I tried to test multiple cases and it works well for most of them, except one:
- Drop a popup
- Drop multiple image snippets such that one could be hidden by the scroll at the bottom
- Scroll to this bottom image and click on it
- without clicking anywhere else, scroll up
- In the image options, click on the crop option
=> the screen is supposed to scroll down to the image and display the cropper, but it is not the case: we don't see the image, nor the cropper
This was the behaviour before 18.0 at least
In general, the cropper behaviour in the popup seems a bit different than before, maybe something else/more to fix ? :slightly_smiling_face:

Reply:
Hmm indeed, I retested and I don't have the issue anymore, strange :open_mouth: Maybe a cache issue I had at that time or something :shrug:  
Let's merge then (after rebasing), given that there is a linked OPW (don't forget to add it in the commit message) :slightly_smiling_face:

Reply:
Alright then, it's rebased. :) Thanks for double-checking.

Reply:
@sobo-odoo maybe if you have the time, could you merge? :) Otherwise I can ask someone else from the team

Reply:
Yes, I just remembered that I forgot to merge it, I was testing it a last time :+1: I will r+ now :slightly_smiling_face: